BH/Modules/Item/ItemDisplay.cpp

```diff
@@ -597,6 +646,12 @@ void Condition::BuildConditions(vector<Condition*> &conditions, string token) {
 	} else if (key.compare(0, 4, "MANA") == 0) {
 		// For unknown reasons, the game's internal Mana stat is 256 for every 1 displayed on item
 		Condition::AddOperand(conditions, new ItemStatCondition(STAT_MAXMANA, 0, operation, value * 256));
+	} else if (key.compare(0, 6, "GOODSK") == 0) {
+		Condition::AddOperand(conditions, new SkillListCondition(operation, 0, value));
```

Comment:
The second param that is used for `type` should be a constant.

Reply:
What do you mean ?

Reply:
The code should be called something like this: `SkillListCondition(operation, CLASS_SKILLS, value)` and `SkillListCondition(operation, CLASS_TAB_SKILLS, value)` then later on in the evaluate function you should test against it like this: `if (type == CLASS_SKILLS)`

Pick constant names that make sense are match other conventions used, those might not be the best ones. The goal here is to make the code more human readable. Assign them in Constants.h